### ISSUE TITLE
runtime error: React component classes must extend React.Component

### ISSUE THREAD

**Issue by rvnath** [eliftech/treeview-react-bootstrap#5]
I am using the treeview like this.

``` javascript
var React = require('react');
var TreeView = require('treeview-react-bootstrap');

  var data = [ {
      text: "Parent 1",
        nodes: [
          {
            text: "Child 1",
            nodes: [
              { text: "Grandchild 1" },
              { text: "Grandchild 2" }
            ]
          },
          { text: "Child 2" }
        ]
    },
    { text: "Parent 2" },
    { text: "Parent 3" },
    { text: "Parent 4" },
    { text: "Parent 5" }
  ];

var ReqTree = React.createClass({
  render: function() {
    var tree = <TreeView data={data} />;
    return tree;
  }
});

module.exports = ReqTree;
```

When I create a barebone app, wih treeview component, it gives me the following errors:
Warning: TreeView(...): React component classes must extend React.Component.warning @ bundle.js:53147ReactCompositeComponentMixin.mountComponent @ bundle.js:39700ReactPerf.measure.wrapper @ bundle.js:46555ReactReconciler.mountComponent @ bundle.js:47183ReactCompositeComponentMixin.mountComponent @ bundle.js:39755ReactPerf.measure.wrapper @ bundle.js:46555ReactReconciler.mountComponent @ bundle.js:47183ReactMultiChild.Mixin.mountChildren @ bundle.js:45886ReactDOMComponent.Mixin._createContentMarkup @ bundle.js:40992ReactDOMComponent.Mixin.mountComponent @ bundle.js:40882ReactReconciler.mountComponent @ bundle.js:47183ReactMultiChild.Mixin.mountChildren @ bundle.js:45886ReactDOMComponent.Mixin._createContentMarkup @ bundle.js:40992ReactDOMComponent.Mixin.mountComponent @ bundle.js:40882ReactReconciler.mountComponent @ bundle.js:47183ReactCompositeComponentMixin.mountComponent @ bundle.js:39755ReactPerf.measure.wrapper @ bundle.js:46555ReactReconciler.mountComponent @ bundle.js:47183ReactCompositeComponentMixin.mountComponent @ bundle.js:39755ReactPerf.measure.wrapper @ bundle.js:46555ReactReconciler.mountComponent @ bundle.js:47183ReactCompositeComponentMixin.mountComponent @ bundle.js:39755ReactPerf.measure.wrapper @ bundle.js:46555ReactReconciler.mountComponent @ bundle.js:47183ReactCompositeComponentMixin.mountComponent @ bundle.js:39755ReactPerf.measure.wrapper @ bundle.js:46555ReactReconciler.mountComponent @ bundle.js:47183mountComponentIntoNode @ bundle.js:45058Mixin.perform @ bundle.js:49846batchedMountComponentIntoNode @ bundle.js:45074Mixin.perform @ bundle.js:49846ReactDefaultBatchingStrategy.batchedUpdates @ bundle.js:42642batchedUpdates @ bundle.js:47916ReactMount._renderNewRootComponent @ bundle.js:45268ReactPerf.measure.wrapper @ bundle.js:46555ReactMount._renderSubtreeIntoContainer @ bundle.js:45342ReactMount.render @ bundle.js:45362ReactPerf.measure.wrapper @ bundle.js:46555(anonymous function) @ bundle.js:76661jQuery.Callbacks.fire @ bundle.js:5255jQuery.Callbacks.self.fireWith @ bundle.js:5367jQuery.extend.ready @ bundle.js:5573completed @ bundle.js:5589
bundle.js:52703 Uncaught Error: Invariant Violation: TreeView.render(): A valid ReactComponent must be returned. You may have returned undefined, an array or some other invalid object.

**Comment by elifTech** [comment]
Thank you for reporting this problem. We are investigating the issue.
We would be grateful if you could specify the version of React you are using.

**Comment by rvnath** [comment]
I am using react version 0.14.

**Comment by elifTech** [comment]
There was the issue due to incompatibility of React versions, because the treeview component had its own react@0.13 as a dependency.
We have decided to move React to peerDependencies.

##### The issue is fixed in version 0.3.0.
